bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -433,6 +433,24 @@ spec:
                 - patch
                 - update
                 - watch
+            - apiGroups:
```

Comment:
These changes should be going to https://github.dev/shubham-pampattiwar/oadp-operator/blob/add-datamover-controller/config/rbac/role.yaml then run `make bundle`

Reply:
Will it fine if I just update the `role.yaml` and not run `make bundle` because it breaks our csv

Reply:
Done updated the PR